charts/vaultwarden/templates/ingress.yaml

```diff
@@ -26,8 +33,7 @@ metadata:
     {{- toYaml $ingress.additionalAnnotations | nindent 4 }}
     {{- end }}
     {{- if $ingress.nginxIngressAnnotations }}
-    nginx.ingress.kubernetes.io/configuration-snippet: |
-      more_set_headers "Request-Id: $req_id";
+    nginx.ingress.kubernetes.io/custom-headers: custom-headers-configmap
```

Comment:
Should this be parameterized so that the ConfigMap can be called anything, but defaulted in `values.yaml`?

Reply:
Done :) I added a new variable named $ingress.customHeaderConfigMap that can be used to define a custom config map name in case someone wants to override it and it defaults to the newly added config map.